portfolio/src/main/webapp/index.html

```diff
@@ -9,16 +9,78 @@
   <body>
     <div id="content">
       <h1>Leah's Portfolio</h1>
-      <p>Hi, I'm Leah. I'm a rising Junior Computer Science student at North Carolina A&T State University, but I grew up in Prince George's County, Maryland. </p>
-      <img src = "/images/me.JPG" alt = "Picture of me :)" width = "400" height = "600">
-      <p>This is my portfolio.</p>
-      <p>Click here for a fact about me:</p>
-      <button onclick="addRandomFact()">Click Here</button>
+      <p>Hi, I'm Leah. I'm a rising Junior Computer Science student at North Carolina A&T State University, 
+          but I grew up in Prince George's County, Maryland. </p>
+      <img src = "/images/me.JPG" alt = "Picture of me :)" width = "450" height = "550" class = "center">
+      <p>This is my personal portfolio.</p>
+      <p>Click below for a fact about me:</p>
+      <button onclick="addRandomFact()">click here</button>
       <div id="fact-container"></div>
       <br>
       <a href="https://www.linkedin.com/in/leahmayo/">Let's connect on LinkedIn!</a>
       <br>
       <a href="https://github.com/leah-mayo">Check out my GitHub!</a>
+      
+      <h2> Comp Sci Courses I've Taken</h2>
+      <p>Intro to Computer Programming</p>
+      <p>Computer Programming Design</p>
+      <p>Discrete Structures</p>
+      <p>Data Structures</p>
+      <p>Analysis of Algorithms</p>
```

Comment:
This seems like it would benefit from being a list. Maybe consider using a unorder list element ul tag and li tags.